src/nfa/delta.cc

```diff
@@ -557,19 +557,26 @@ OrdVector<Symbol> Delta::get_used_symbols() const {
     // return ov;
 
     ///WITH BOOL VECTOR, DIFFERENT VARIANT? (1.9s):
-    //std::vector<bool> bv = get_used_symbols_bv();
-    //std::vector<Symbol> v(std::count(bv.begin(), bv.end(), true));
-    //return utils::OrdVector<Symbol>(v);
+    // std::vector<bool> bv = get_used_symbols_bv();
+    // utils::OrdVector<Symbol> ov{};
+    // ov.reserve(static_cast<size_t>(std::count(bv.begin(), bv.end(), true)));
+    // const size_t bv_size{ bv.size() };
+    // for (Symbol i = 0; i < bv_size; i++) {
+    //     if (bv[i]) {
+    //         ov.push_back(i);
+    //     }
+    // }
+    // return ov;
```

Comment:
What happened here?

Reply:
The `bv` was computed, but never properly utilized. And `v` was never initialized properly with correct values, only preallocated. This change fixes this variant so that it can be uncommented and used again.